### ISSUE TITLE
No history available when running docker history on a Jib build docker image

### ISSUE THREAD

**Issue by anthonydahanne** [GoogleContainerTools/jib#840]
**Description of the issue**:
I've create a docker image with jib, anthonydahanne/petclinic:jib ; when I run docker history on it, there is no layer described

**Expected behavior**:
See layers description / metadata added at each step

**Steps to reproduce**: 
```
[node3] (local) root@192.168.0.41 ~
$ docker pull anthonydahanne/petclinic:jib
jib: Pulling from anthonydahanne/petclinic
8f125ded1b48: Pull complete
ba7c544469e5: Pull complete
43b4357066b0: Pull complete
4dcd5ac6fd80: Pull complete
16a5fb589b85: Pull complete
d24512140230: Pull complete
Digest: sha256:cf8716fddcce10c93c9803562acf2dd48804983e4d5ee6a6db151930d84df697
Status: Downloaded newer image for anthonydahanne/petclinic:jib
[node3] (local) root@192.168.0.41 ~
$ docker history anthonydahanne/petclinic:jib
IMAGE               CREATED             CREATED BY          SIZE                COMMENT

```
I would have expected at least 6 layers to show up, since, after all, docker pulled 6 layers

**Environment**: 
Docker version 18.03.1-ce, build 9ee9f40

**`jib-maven-plugin` Configuration**: 
```xml
     <plugin>
        <groupId>com.google.cloud.tools</groupId>
        <artifactId>jib-maven-plugin</artifactId>
        <version>0.9.8</version>
        <configuration>
          <to>
            <image>registry.hub.docker.com/anthonydahanne/petclinic:jib</image>
          </to>
        </configuration>
      </plugin>
```

**Comment by TadCordle** [comment]
This is currently being tracked by #875, going to close this issue in favor of that one.

**Comment by anthonydahanne** [comment]
okee - thanks

**Comment by coollog** [comment]
@anthonydahanne We have released version `0.9.10` that writes history for the Docker image.

**Comment by anthonydahanne** [comment]
thanks @coollog for letting me know.
I've just tested, and I confirm 0.9.10 provides history : 
```
docker history anthonydahanne/petclinic:jib --no-trunc
IMAGE                                                                     CREATED             CREATED BY          SIZE                COMMENT
sha256:1ed95add60958bd72e99b428c5b43b8b397272d08e545668725179f0273bafb1   48 years ago        jib-maven-plugin    47.8kB
<missing>                                                                 48 years ago        jib-maven-plugin    940kB
<missing>                                                                 48 years ago        jib-maven-plugin    38.4MB
<missing>                                                                 48 years ago        bazel build ...     99.8MB
<missing>                                                                 48 years ago        bazel build ...     1.93MB
<missing>                                                                 48 years ago        bazel build ...     16.6MB
```
